azext_iot/central/params.py

```diff
@@ -150,6 +150,13 @@ def load_central_arguments(self, _):
             " A case-sensitive string (up to 128 characters long) of ASCII 7-bit alphanumeric characters plus"
             " certain special characters: - . + % _ # * ? ! ( ) , : = @ $ '",
         )
+    with self.argument_context("iot central device manual-failover") as context:
+        context.argument(
+            "ttl_minutes",
+            type=int,
+            options_list=["--ttl-minutes","--ttl"],
+            help="A positive integer. TTL in minutes to move device back to the original hub. Defaults to 30 minutes if not specified.",      
```

Comment:
if this can change async in the backend then remove the defaults to statement or CLI users might get confused